confignode/src/main/java/org/apache/iotdb/confignode/physical/crud/CreateRegionsPlan.java

```diff
@@ -55,24 +47,35 @@ public void setStorageGroup(String storageGroup) {
     this.storageGroup = storageGroup;
   }
 
-  public Map<Integer, List<Integer>> getDeviceGroupIDs() {
-    return deviceGroupStartTimeMap;
+  public void addRegion(RegionReplicaSet regionReplicaSet) {
+    this.regionReplicaSets.add(regionReplicaSet);
   }
 
-  public void setDeviceGroupIDs(Map<Integer, List<Integer>> deviceGroupIDs) {
-    this.deviceGroupStartTimeMap = deviceGroupIDs;
+  public List<RegionReplicaSet> getRegionReplicaSets() {
+    return regionReplicaSets;
   }
 
   @Override
   protected void serializeImpl(ByteBuffer buffer) {
```

Comment:
As a witness to stand-alone modules, we had better add UT to all plan serialization and deserialization methods, otherwise it is very error-prone here

Reply:
Added tests.